src/pngimage.cpp

```diff
@@ -561,16 +581,21 @@ namespace Exiv2 {
             memcpy(szChunk,cheaderBuf.pData_ + 4,4);
             szChunk[4]  = 0;
 
-            if (!memcmp(cheaderBuf.pData_ + 4, "IEND", 4))
+            if ( !strcmp(szChunk,"IEND") )
```

Comment:
👍 for replacing the `memcmp` functions by `strcmp`

Reply:
Would you like me to revert some of this stuff and update the PR?

Reply:
nono, the comment was positive, I think it is more readable to use `strcmp` for the string comparisons rather than the original `memcp` 😉